rke/k8s_service_options.go

```diff
@@ -156,6 +156,14 @@ func loadK8sVersionServiceOptions() map[string]v3.KubernetesServicesOptions {
 			Kubeproxy:      getKubeProxyOptions(),
 			Scheduler:      getSchedulerOptions(),
 		},
+		"v1.15.12-rancher2-3": {
```

Comment:
How come this doesn't exist for the other new versions in 1.16, 1.17, 1.18?

Reply:
added for 1.16, 1.17 and 1.18 fallback to default